modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Activities/components/Listing/Listing.tsx

```diff
@@ -46,9 +50,10 @@ const Listing = ({
 					activities.map((activity, index) => (
 						<ActivityPanel
 							activity={activity}
+							hasEmptyFields={Boolean(errors.activities?.[index])}
```

Comment:
```suggestion
							hasErrors={hasActivityErrorsByIndex(index)}
```

Acredito que seja melhor passar uma função ao inves do valor do erros inteiro para nao gerar uma dependencia.